### ISSUE TITLE
Angular / web components support

### ISSUE THREAD

**Issue by BorntraegerMarc** [angular/angular#14350]
I'm interested on how angular plans to support native web components in the future - as there has been no official statement.

I didn't get an answer here: https://stackoverflow.com/questions/41590964/angular-integration-with-web-components-roadmap

That's why I'm posting it again on GitHub (sorry I know it's wrong).

**Comment by FrozenPandaz** [comment]
Custom Elements/ Web Components are 1st class citizens of HTML once they are defined to the browser.

Angular supports them as much as it does a ```<h1>``` or ```<a>```.

**Comment by BorntraegerMarc** [comment]
@FrozenPandaz I totally agree that this Issue Tracker is not for Support Requests, but I was kinda despreate because I did not get an answer anywhere...

Thanks for the article! A bit more specific: I has google polymer in mind. Would that mean that Polymer 2.0 would be fully supported by Angular from the ground up because the new version uses purely web standards?

Because I noticed that polymer is not usable without a 3rd party lib like https://github.com/platosha/angular-polymer...
